submissions/FoxYulya/HTML_CSS_Practice/style.css

```diff
@@ -0,0 +1,182 @@
+:root {
+    --bg-color: rgb(221, 215, 178);
+    --border-color: rgb(221, 215, 178);
+    --text-color: rgb(41, 37, 47);
+    --show-more: rgb(201, 219, 189);
+  }
+  
+  * {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    transition: 0.1s;
+  }
+  
+  
+  .wrapper {
+    max-width: 1200px;
+    padding: 0 15px;
+    margin: 0 auto;
+  }
+  
+  .header {
+    display: flex;
+    justify-content: flex-end;
+    margin: 15px 0;
+    padding: 15px;
+    height: 54px;
+    border-radius: 8px;
+    border: 2px solid var(--border-color);
+    background-color: var(--bg-color);
+    position: relative;
+  }
+  /* li menu --------------------------- */
+
+  .main-menu > ul >li {
+    display: inline-block;  
+  }
+
+  header .wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 100%;
+  }
+
+  .main-menu > ul > li > a {
```

Comment:
Avoid to use too specific selectors, read more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)